src/Composer/Command/InitCommand.php

```diff
@@ -487,15 +487,6 @@ public function parseAuthorString($author)
         );
     }
 
-    /**
-     * @param string $name
-     * @return list<array{name: string, description: ?string}>
-     */
-    protected function findPackages($name)
```

Comment:
Is this internal enough to be removed? If not the return type of `RepositoryInterface::search` simply needs to be duplicated

Reply:
I guess time will tell but I'd say it's fine.